src/e2e-test/java/io/cdap/cdap/ui/utils/Constants.java

```diff
@@ -24,8 +24,8 @@ private static double getRandomArbitrary(int min, int max) {
   public static final int TEST_TIMEOUT_TIME = 10000;
   public static final int RETRY_INTERVAL = 2000;
   public static final String TEST_TIMEOUT_MESSAGE = "Timed out after" +
-    String.valueOf(TEST_TIMEOUT_TIME / 1000) +
-    "seconds";
+      String.valueOf(TEST_TIMEOUT_TIME / 1000) +
```

Comment:
Please don't change this indentation

Reply:
Sure.

Reply:
This is still not addressed, not sure why it's resolved.